src/overlay_params.cpp

```diff
@@ -160,8 +161,33 @@ parse_string_to_keysym_vec(const char *str)
 #define parse_upload_log(x)            {}
 #define parse_upload_logs(x)           {}
 #define parse_toggle_fps_limit(x)      {}
+#define parse_toggle_preset(x)         {}
 #endif
 
+// NOTE: This is NOT defined as an OVERLAY_PARAM and will be called manually
+static std::vector<int>
+parse_preset(const char *str)
```

Comment:
Why is this function necessary and why can't we use the normal params methods?

Reply:
I didn't use the normal params methods because the a parameters specified in a preset should be set before any other parameters are set.  This is how presets work today